scripts/zones/Newton_Movalpolos/npcs/Furnace_Hatch.lua

```diff
@@ -9,13 +9,12 @@ require("scripts/globals/status")
 
 function onTrade(player, npc, trade)
     if npcUtil.tradeHas(trade, 947) then
-        local offset = npc:getID() - ID.npc.FURNACE_HATCH_OFFSET
```

Comment:
offset is still needed to play the correct event (I think it's a short cutscene of your character toggling the lever) on line 13